src/sampling.jl

```diff
@@ -0,0 +1,50 @@
+import StatsBase: sample
+using Random, KernelDensity
+
+function sample(rng::AbstractRNG, chn::MCMCChains.AbstractChains, n;
+    replace=true, ordered=false) 
```

Comment:
Some comments on the meaning of `replace` and `ordered` might be helpful here.

Reply:
True!

In this particular instance, ```sample()```,  StatsBase will step in:
```
? sample
```
will return:
```
help?> sample
search: sample sampler sample! samplepair Sampleable wsample wsample! nsamples

  sample([rng], a, [wv::AbstractWeights], n::Integer; replace=true, ordered=false)

  Select a random, optionally weighted sample of size n from an array a using a
  polyalgorithm. Sampling probabilities are proportional to the weights given in
  wv, if provided. replace dictates whether sampling is performed with replacement
  and order dictates whether an ordered sample, also called a sequential sample,
  should be taken.


  Optionally specify a random number generator rng as the first argument (defaults
  to Random.GLOBAL_RNG).
```

but in general, MCMCChains has enough how-to-use-it and features that it warrants it's own docs.